src/v2/Components/Artwork/FillwidthItem.tsx

```diff
@@ -57,20 +58,15 @@ export class FillwidthItemContainer extends React.Component<
       },
     } = this.props
 
-    const imageWidth = Math.floor(this.imageHeight * aspect_ratio)
+    const imageWidth = Math.floor(this.props.imageHeight * aspect_ratio)
     return imageWidth
   }
 
   get imageHeight() {
-    // During the SSR render pass we don't have access to window pixel data so
-    // default to high density screen.
-    const devicePixelRatio =
-      typeof window === "undefined" ? 2 : window.devicePixelRatio
-
-    return this.props.imageHeight * devicePixelRatio
```

Comment:
So... this almost certainly has some performance implications. Basically we've been loading both the 1x and 2x versions of every thumbnail in a fillwidth rail. 🖼️ 🖼️ 

Going to see if I can dig up some metrics.